src/renderers/dom/fiber/wrappers/ReactDOMFiberInput.js

```diff
@@ -97,6 +98,39 @@ var ReactDOMInput = {
         getCurrentFiberStackAddendum,
       );
 
+      if (props.type === 'checkbox') {
```

Comment:
I don't quite understand this warning. I think the one below should be enough. Maybe this is a merge conflict of some sort?